core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/services/html/dot-seo-meta-tags.service.ts

```diff
@@ -53,13 +53,27 @@ export class DotSeoMetaTagsService {
         const title = pageDocument.querySelectorAll('title');
         const titleOgElements = pageDocument.querySelectorAll('meta[property="og:title"]');
         const imagesOgElements = pageDocument.querySelectorAll('meta[property="og:image"]');
+        const descriptionOgElements = pageDocument.querySelectorAll(
+            'meta[property="og:description"]'
+        );
+        const twitterCardElements = pageDocument.querySelectorAll('meta[name="twitter:card"]');
+        const twitterTitleElements = pageDocument.querySelectorAll('meta[name="twitter:title"]');
+        const twitterImageElements = pageDocument.querySelectorAll('meta[name="twitter:image"]');
+        const twitterDescriptionElements = pageDocument.querySelectorAll(
+            'meta[name="twitter:description"]'
+        );
 
         metaTagsObject['faviconElements'] = favicon;
         metaTagsObject['titleElements'] = title;
         metaTagsObject['favicon'] = (favicon[0] as HTMLLinkElement)?.href;
         metaTagsObject['title'] = title[0]?.innerText;
         metaTagsObject['titleOgElements'] = titleOgElements;
         metaTagsObject['imageOgElements'] = imagesOgElements;
+        metaTagsObject['twitterCardElements'] = twitterCardElements;
+        metaTagsObject['twitterTitleElements'] = twitterTitleElements;
+        metaTagsObject['twitterDescriptionElements'] = twitterDescriptionElements;
+        metaTagsObject['twitterImageElements'] = twitterImageElements;
+        metaTagsObject['descriptionOgElements'] = descriptionOgElements;
```

Comment:
Does this `metaTagsObject` have a interface?

Reply:
Yes it does: export interface SeoMetaTags